### PR TITLE
using version 5 of forked goredis to fix the build

### PR DESCRIPTION
**Pull request by mohandutt134**

### CODE REVIEW

File: backends/pool.go [hunk 1]
```diff
@@ -5,7 +5,7 @@ import (
 	"sync"
 	"time"
 
-	"gopkg.in/redis.v4"
```
Comment (meson10) [inline backends/pool.go:8]: Why call it v5? The repo has been changed anyway. Keep it as 4.

File: backends/redis.go [hunk 2]
```diff
@@ -8,7 +8,7 @@ import (
 	"time"
 
 	"gopkg.in/gilmour-libs/gilmour-e-go.v4/protocol"
-	"gopkg.in/redis.v4"
```
Comment (meson10) [inline backends/redis.go:11]: Again, a fork of 4 should be at 4 only no?  Although it doesn't matter to be honest, but logically it sounds so incorrect :)

File: .travis.yml [hunk 3]
```diff
@@ -7,6 +7,10 @@ services:
   - redis-server
 
 script:
+  - rm -rf $GOPATH/src/gopkg.in/gilmour-libs/gilmour-e-go.v4/backends
+  - rm -rf $GOPATH/src/gopkg.in/gilmour-libs/gilmour-e-go.v4/ui
```
Comment (meson10) [inline .travis.yml]: @mohandutt134 Won't this fail? What if $GOPATH/src/gopkg.in/gilmour-libs/gilmour-e-go.v4/ doesn't exist, which in most likelihood might not be there.
Maybe you should create that directory path first?

File: .travis.yml [hunk 4]
```diff
@@ -7,6 +7,10 @@ services:
...
+  - ls -s $(pwd)/backends $GOPATH/src/gopkg.in/gilmour-libs/gilmour-e-go.v4/backends
```
Comment (meson10) [inline .travis.yml]: Also, I guess you wanted to do `ln -s` and not `ls -s`

File: backends/pool.go [hunk 5]
```diff
@@ -5,7 +5,7 @@ import (
...
+	"gopkg.in/mohandutt134/redis.v5"
```
Comment (meson10) [inline backends/pool.go]: Tangent suggestion: You should rename your username to `mohandutt1234`

File: .travis.yml [hunk 6]
```diff
@@ -7,6 +7,9 @@ services:
   - redis-server
 
 script:
+  - mkdir -p $GOPATH/src/gopkg.in/gilmour-libs/gilmour-e-go.v4
+  - ls -s $(pwd)/backends $GOPATH/src/gopkg.in/gilmour-libs/gilmour-e-go.v4/backends
```
Comment (meson10) [inline .travis.yml]: How does this work?

Comment (meson10) [inline .travis.yml]: Should not it be `ln -s` instead of `ls -s`?

Comment (mohandutt134) [inline .travis.yml]: @meson10 But this is working on travis, I confirmed it.
source: http://askubuntu.com/questions/56339/how-to-create-a-soft-or-symbolic-link

**Pull request merged**